torch/nn/functional.py

```diff
@@ -2611,15 +2638,11 @@ def gaussian_nll_loss(input, target, var, *, full=False, eps=1e-6, reduction='me
     with torch.no_grad():
         var.clamp_(min=eps)
 
-    # Calculate loss (without constant)
-    loss = 0.5 * (torch.log(var) + (input - target)**2 / var).view(input.size(0), -1).sum(dim=1)
-
-    # Add constant to loss term if required
+    # Calculate the loss
+    loss = 0.5 * (torch.log(var) + (input - target)**2 / var)
```

Comment:
This is BC-breaking.
Can you please add a comment in the PR description that explains what was the previous behavior, the new one and why we made that change. As well as a code example on how to recover the old behavior.

cc @jbschlosser This was added fairly recently so I guess we're happy to be BC-breaking here? Maybe we should confirm that in our next sync though?

Reply:
@albanD Since this loss was just added in 1.8.0, I'm good with the BC-breaking change here to bring the reduction behavior in line with that of other losses.

Reply:
Thanks both, will update the PR description.

Reply:
I've updated the PR description, let me know if there's anything else to add.

Reply:
The description looks great!